src/main/java/com/rachelbock/resources/ProjectResource.java

```diff
@@ -0,0 +1,127 @@
+package com.rachelbock.resources;
+
+import com.rachelbock.data.Climb;
+import com.rachelbock.data.Project;
+import com.rachelbock.data.User;
+
+import javax.swing.plaf.nimbus.State;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Properties;
+
+/**
+ * Class to set up the database connection for projects
+ */
+
+@Path("/projects/{user_name}")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ProjectResource {
+
+    public Connection getConnection() throws SQLException {
```

Comment:
Probably getting to the point where we should talk about using a connection pool, that can wait for the next PR though